### PR TITLE
add imsanti results

### PR DESCRIPTION
**Pull request by santiaghini**
Optimized hyperparameters (lots of sweeping) and replaced default FFN for a SwiGLU based FFN. Achieved validation loss of 3.79 on OpenWebText.

Link: https://api.wandb.ai/links/estanfor/6ia1n3zv

### CODE REVIEW

**Comment by nelson-liu** [comment]
thanks!
